app/models/order.rb

```diff
@@ -0,0 +1,22 @@
+class Order < ApplicationRecord
+  belongs_to :user
+  has_many :order_outings
+  has_many :outings, through: :order_outings
+
+  enum status: %w(ordered paid cancelled shipped completed)
+
+  scope :ordered, -> { where(status: "ordered") }
+  scope :paid, -> { where(status: "paid") }
+  scope :cancelled, -> { where(status: "cancelled") }
+  scope :completed, -> { where(status: "completed") }
```

Comment:
You shouldn't need these scopes. Because you are using `enum`, you automatically get those scopes that you can call on the model. http://api.rubyonrails.org/classes/ActiveRecord/Enum.html